DEBUG:root:Start polling
INFO:aiogram.dispatcher:Start polling
INFO:aiogram.dispatcher:Run polling for bot @cyberpunk_pager_bot id=6476060155 - 'cyberpunk_pager'
DEBUG:root:Set state: states.RegisterState.number_group, data: Зарегистрироваться
INFO:aiogram.event:Update id=505770854 is handled. Duration 202 ms by bot id=6476060155
DEBUG:root:Set state: states.RegisterState.number_group, data: Зарегистрироваться
INFO:aiogram.event:Update id=505770853 is handled. Duration 216 ms by bot id=6476060155
DEBUG:root:Set state: states.RegisterState.number_group, data: Зарегистрироваться
INFO:aiogram.event:Update id=505770855 is handled. Duration 235 ms by bot id=6476060155
DEBUG:root:Set state: states.RegisterState.number_group, data: Зарегистрироваться
INFO:aiogram.event:Update id=505770856 is handled. Duration 264 ms by bot id=6476060155
DEBUG:root:Set state: states.RegisterState.number_group, data: Зарегистрироваться
INFO:aiogram.event:Update id=505770857 is handled. Duration 158 ms by bot id=6476060155
DEBUG:root:Set state: states.RegisterState.number_group, data: Зарегистрироваться
INFO:aiogram.event:Update id=505770858 is handled. Duration 105 ms by bot id=6476060155
DEBUG:root:Set state: states.RegisterState.number_group, data: Зарегистрироваться
INFO:aiogram.event:Update id=505770859 is handled. Duration 83 ms by bot id=6476060155
DEBUG:root:Set state: states.RegisterState.number_group, data: Зарегистрироваться
INFO:aiogram.event:Update id=505770860 is handled. Duration 179 ms by bot id=6476060155
INFO:aiogram.event:Update id=505770861 is handled. Duration 183 ms by bot id=6476060155
INFO:aiogram.event:Update id=505770862 is not handled. Duration 1 ms by bot id=6476060155
ERROR:aiogram.event:Cause exception while process update id=505770862 by bot id=6476060155
ValueError: invalid literal for int() with base 10: 'ппп'
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
  File "/usr/local/lib/python3.10/dist-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
  File "/usr/local/lib/python3.10/dist-packages/aiogram/dispatcher/middlewares/user_context.py", line 49, in __call__
    return await handler(event, data)
  File "/usr/local/lib/python3.10/dist-packages/aiogram/fsm/middleware.py", line 42, in __call__
    return await handler(event, data)
  File "/usr/local/lib/python3.10/dist-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "/usr/local/lib/python3.10/dist-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
  File "/usr/local/lib/python3.10/dist-packages/aiogram/dispatcher/dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/aiogram/dispatcher/router.py", line 142, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "/usr/local/lib/python3.10/dist-packages/aiogram/dispatcher/router.py", line 137, in _wrapped
    return await self._propagate_event(
  File "/usr/local/lib/python3.10/dist-packages/aiogram/dispatcher/router.py", line 170, in _propagate_event
    response = await router.propagate_event(update_type=update_type, event=event, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/aiogram/dispatcher/router.py", line 142, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
  File "/usr/local/lib/python3.10/dist-packages/aiogram/dispatcher/router.py", line 137, in _wrapped
    return await self._propagate_event(
  File "/usr/local/lib/python3.10/dist-packages/aiogram/dispatcher/router.py", line 162, in _propagate_event
    response = await observer.trigger(event, **kwargs)
  File "/usr/local/lib/python3.10/dist-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
  File "/usr/local/lib/python3.10/dist-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
  File "/root/workspace/Cyberpunk_Pager_Bot/src/pager/handlers/register.py", line 36, in cmd_register_nickname
    new_player.number_group = int(message.text)
ValueError: invalid literal for int() with base 10: 'ппп'
INFO:aiogram.event:Update id=505770863 is handled. Duration 222 ms by bot id=6476060155
DEBUG:root:Set state: states.RegisterState.number_group, data: Зарегистрироваться
INFO:aiogram.event:Update id=505770864 is handled. Duration 106 ms by bot id=6476060155
DEBUG:root:Set state: states.RegisterState.nickname, data: 1
INFO:aiogram.event:Update id=505770865 is handled. Duration 75 ms by bot id=6476060155
DEBUG:root:Set state: states.RegisterState.done. data: {'number_group': '1', 'nickname': 'Вика'}
INFO:aiogram.event:Update id=505770866 is handled. Duration 91 ms by bot id=6476060155
WARNING:aiogram.dispatcher:Received SIGTERM signal
INFO:aiogram.dispatcher:Polling stopped for bot @cyberpunk_pager_bot id=6476060155 - 'cyberpunk_pager'
INFO:aiogram.dispatcher:Polling stopped
